evaluate/evaluate_test.go

```diff
@@ -535,11 +559,12 @@ func TestEvaluate(t *testing.T) {
 					Runs: 2,
 				},
 
-				ExpectedAssessments: map[evalmodel.Model]map[language.Language]map[string]metrics.Assessments{
-					mockedModel: map[language.Language]map[string]metrics.Assessments{
-						languageGolang: map[string]metrics.Assessments{
-							repositoryPlainPath: map[metrics.AssessmentKey]uint64{},
-							repositoryNextPath:  map[metrics.AssessmentKey]uint64{},
+				ExpectedAssessments: map[evalmodel.Model]map[language.Language]map[string]map[task.Identifier]metrics.Assessments{
+					mockedModel: map[language.Language]map[string]map[task.Identifier]metrics.Assessments{
+						languageGolang: map[string]map[task.Identifier]metrics.Assessments{
+							repositoryPlainPath: map[task.Identifier]metrics.Assessments{
+								task.IdentifierWriteTests: map[metrics.AssessmentKey]uint64{},
+							},
 						},
```

Comment:
`repositoryNextPath` is gone. Guess does not matter if its empty or gone, but I think we should keep it?

Reply:
Why do you think we should keep it?

The test is the following: "Never solving basic checks leads to exclusion". So this is IMO fine if there is nothing for `repositoryNextPath`.

Also the reason why it was there before is because the nested maps were preinitialized for all models, languages and repositories. We would need to do the same for tasks. But not every model supports every task, and not every repository allows all tasks. So this gets complicated to preinitialize. It's way easier to initialize on demand. So unless there is a good reason why we need the old version, I'd keep  this one.